frontend/packages/component-library/src/cms/editorialCard/stories/EditorialCard.story.tsx

```diff
@@ -0,0 +1,90 @@
+import imageFile from '@public/images/image-component.png';
+import type {Meta, StoryObj} from '@storybook/react';
+
+import EditorialCard, {
+  EDITORIAL_CARD_LAYOUTS,
+  EditorialCardProps,
+} from './../EditorialCard';
+
+type Story = StoryObj<EditorialCardProps | EditorialCardProps[]>;
+
+export default {
+  title: 'CMS/EditorialCard',
+  component: EditorialCard,
+  render: args => {
+    const components: EditorialCardProps[] = args[0]
+      ? Object.values(args)
+      : [args];
+    return (
+      <>
+        {components.map((component, index) => (
+          <section key={index} style={{width: '27rem', marginBottom: '2rem'}}>
+            <EditorialCard {...defaultProps} {...component} />
+          </section>
+        ))}
+      </>
+    );
+  },
+} as Meta;
+
+const defaultProps: EditorialCardProps = {
+  heading: 'EditorialCard Heading',
+  text:
+    'Students can explore their imagination as ' +
+    'they design unique, interactive experiences,\n' +
+    'making game design a powerful tool for ' +
+    'artistic expression.',
```

Comment:
Can you make this one string w/o concatenation? It's causing weird line breaks in the vertical layout.

Reply:
What do you mean?

Reply:
It's showing like this on the vertical card. The width of these cards should be handled with css so it's dynamic, not reliant on the text being broken up manually. 
<img width="733" alt="Screenshot 2025-04-21 at 10 33 27 AM" src="https://github.com/user-attachments/assets/268cff2b-47b4-4815-8cc9-56b9a427f46a" />


```suggestion
    'Students can explore their imagination as they design unique, interactive experiences, making game design a powerful tool for artistic expression.',
```

Reply:
If you are referring to the line break, I just wanted to highlight that it shows the component does support multiline text. The string concatenation does not affect this at all — it is only there to make the code easier to read and to show the text the way it naturally appears within the given width

Reply:
Yes, but you can do that using a non-concatenated sentence which is a more real-life use case.

Reply:
We don't usually have multi-line text here so I don't think it's necessary to show this.